yarn.lock

```diff
@@ -2099,11 +2099,6 @@ abab@^2.0.0:
   resolved "https://registry.yarnpkg.com/abab/-/abab-2.0.3.tgz#623e2075e02eb2d3f2475e49f99c91846467907a"
   integrity sha512-tsFzPpcttalNjFBCFMqsKYQcWxxen1pgJR56by//QwvJc4/OUS3kPOOttx2tSIfjsylB0pYu7f5D3K1RCxUnUg==
 
-abbrev@1:
```

Comment:
I don't understand why yarn.lock has so many diffs.  I get something similar if I just run `yarn install` on the develop branch...  Is it not deterministic or something? :-/

Reply:
probably everyone running different node versions

Reply:
I wondered if that might be it.  I kinda' hoped it wasn't though. 😄  I'm on node 12.16.0 and yarn 1.21.1.  If you know of a "better" one, I can switch and add it to the onboarding doc...